ci_framework/roles/devscripts/tasks/sub_tasks/16_user.yml

```diff
@@ -0,0 +1,57 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create the specified user.
+  become: true
+  ansible.builtin.user:
+    name: "{{ cifmw_devscripts_user }}"
+    state: present
+
+- name: Ensure the devscripts required directories exists.
+  become: true
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: "directory"
+    owner: "{{ cifmw_devscripts_user }}"
+    group: "{{ cifmw_devscripts_user }}"
+    mode: "0755"
+  loop:
+    - "{{ cifmw_devscripts_repo_dir }}"
+    - "{{ cifmw_devscripts_config['working_dir'] }}"
+
+- name: Generate SSH keys for accessing OCP cluster
+  community.crypto.openssh_keypair:
```

Comment:
(note: it's an already installed collection - no need to look for its addition in the /requirements.yml ;) )